src/ore/Cargo.toml

```diff
@@ -59,6 +61,7 @@ hyper-tls = { version = "0.5.0", optional = true }
 opentelemetry = { git = "https://github.com/MaterializeInc/opentelemetry-rust.git", features = ["rt-tokio", "trace"], optional = true }
 opentelemetry-otlp = { git = "https://github.com/MaterializeInc/opentelemetry-rust.git", optional = true }
 console-subscriber = { version = "0.1.8", optional = true }
+sentry-tracing = "0.27.0"
 
 [dev-dependencies]
 tokio = { version = "1.19.2", features = ["macros"] }
```

Comment:
Can't leave a line comment on it, but you'll need to add `sentry` and `sentry-tracing` to the `tracing_` feature.